test/optimizations/localon/recursive-local-on.good

```diff
@@ -1,2 +1,2 @@
 {x = 5}
-{x = recursive-local-on.chpl:11: error: cannot access remote data in local block
+{recursive-local-on.chpl:11: error: cannot access remote data in local block
```

Comment:
Same Q here

Reply:
In the serialization implementation it appears that there is a check for locality before the "x =" is printed, compared to the old writeThis implementation.